packages/brand/Kaizen/Brand/Brand.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { assetUrl } from "../../../hosted-assets"
```

Comment:
Think you'll want to import from `"@kaizen/hosted-assets"` instead and add that package as a dependency.